endpoint.go

```diff
@@ -1109,3 +1109,26 @@ func (c *controller) cleanupLocalEndpoints() {
 		}
 	}
 }
+
+func (ep *endpoint) setAliasIP(sb *sandbox, ip net.IP, add bool) error {
+	sb.Lock()
+	sbox := sb.osSbox
+	sb.Unlock()
+
+	for _, i := range sbox.Info().Interfaces() {
+		if ep.hasInterface(i.SrcName()) {
+			var (
+				ipNet *net.IPNet
+				err   error
+			)
+			if _, ipNet, err = net.ParseCIDR(ip.String() + "/32"); err != nil {
+				return err
+			}
```

Comment:
You do not need to do this string concatenation and then parsing, just construct the CIDR yourself with:
```
if err := i.SetAliasIP(&net.IPNet{IP: ip Mask: net.CIDRMask(32, 32)}, add); ...
```